samwise.js

```diff
@@ -1,33 +1,33 @@
-const uniqueSortedArray = [...new Set(array)].sort();
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.